app-arch/zopfli/zopfli-1.0.3.recipe

```diff
@@ -31,17 +32,23 @@ BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	"
 BUILD_PREREQUIRES="
+	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
 	cmd:make
 	"
 
 BUILD()
 {
+	mkdir -p build && cd build
+	cmake .. \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX="$commandBinDir"
 	make $jobArgs
 }
 
 INSTALL()
 {
 	install -d "$commandBinDir"
 	install -t "$commandBinDir" zopfli
+	install -t "$commandBinDir" zopflipng
```

Comment:
Didn't check, but isn't there a "make install"?

Reply:
@Begasus When I try to do "make install" it says there is no rule to make target "install".

Reply:
You probably want to change directory for that first :)
```diff
+	cd build
+	make install
```

Reply:
Ok, nevermind. Now should be good I think.